whois/parser.py

```diff
@@ -272,26 +278,44 @@ def load(domain, text):
             return WhoisIe(domain, text)
         elif domain.endswith('.nz'):
             return WhoisNz(domain, text)
-        elif domain.endswith('.space'):
+        elif domain.endswith('.space') or domain.endswith('.bz'):
             return WhoisSpace(domain, text)
         elif domain.endswith('.lu'):
             return WhoisLu(domain, text)
         elif domain.endswith('.cz'):
             return WhoisCz(domain, text)
         elif domain.endswith('.online'):
             return WhoisOnline(domain, text)
+        elif domain.endswith('.site') or domain.endswith('.website'):
+            return WhoisIANA(domain, text)
         else:
             return WhoisEntry(domain, text)
 
 
 class WhoisSpace(WhoisEntry):
     """Whois parser for .space domains
     """
+    regex = {
+        'whois_server': 'whois: *(.+)',
+        'updated_date': 'changed: *(.+)',
+        'creation_date': 'created: *(.+)',
+        'name_servers': 'nserver: *(.+)',  # list of name servers
+        'status': 'status: *(.+)',   # list of statuses
+        'emails': EMAIL_REGEX,   # list of email s
+        'address': 'address: *(.+)'
+    }
+
     def __init__(self, domain, text):
         if 'No match for "' in text:
             raise DomainNotFoundError(text)
         else:
-            WhoisEntry.__init__(self, domain, text)
+            WhoisEntry.__init__(self, domain, text, self.regex)
```

Comment:
Thoughts on this (and ditto the same thing further down in `WhoisIANA`):
1. use `super()`: `super(WhoisSpace, self).__init__(domain, text, self.regex)`
2. drop the `else:` - the if clause blows up execution anyway, so you can lose the `else:` and save a little bit on overall complexity

It should work fine as written, but using `super()` is the correct way to call a superclass method; dropping the `else` is just a stylistic issue, up to you

Reply:
That's actually an artifact of the original design (aka not my design), but I can update it if you want. For every class. Uhhh.

Reply:
Then file under "hellllll nope" - let's leave it uniformly shitty :)